src/Form.js

```diff
@@ -0,0 +1,45 @@
+import firebase from './firebase.js';
+import { useState } from 'react';
+
+const Form = () => {
+    const [ taskName, setTask ] = useState('');
+    const handleOnChange = (e) => {
+        setTask(e.target.value);
+    }
+
+    const addTask = () => {
+        const taskRef = firebase.database().ref('Task');
+        const task = {
+            taskName,
+            complete: false,
+        };
+        taskRef.push(task);
+        setTask('');
+    }
+
+    return (
+        <div className='form'>
+            <form>
+                <label htmlFor="">Task Name</label>
```

Comment:
You can pair labels and their associated inputs by giving the same value to htmlFor="" in label  and id="" in input. This also helps with accessibility because the browser/SRs will understand they're connected!